lib/plugins/virtAsstBase.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+var moment = require('moment');
+var _each = require('lodash/each');
+
+function init(env, ctx) {
+  function virtAsstBase() {
+    return virtAsstBase;
+  }
+
+  var entries = ctx.entries;
+  var translate = ctx.language.translate;
+
+  virtAsstBase.setupMutualIntents = function (configuredPlugin) {
+    // full status
+    configuredPlugin.addToRollup('Status', function (slots, sbx, callback) {
+      entries.list({count: 1}, function (err, records) {
+        var direction;
+        if (translate(records[0].direction)) {
+          direction = translate(records[0].direction);
+        } else {
+          direction = records[0].direction;
+        }
+        var status = translate('virtAsstStatus', {
+          params: [
+            sbx.scaleMgdl(records[0].sgv),
+            direction,
+            moment(records[0].date).from(moment(sbx.time))
+          ]
+        });
+        
+        callback(null, {results: status, priority: -1});
+      });
+    }, 'BG Status');
+
+    configuredPlugin.configureIntentHandler('NSStatus', function (callback, slots, sbx, locale) {
+      configuredPlugin.getRollup('Status', sbx, slots, locale, function (status) {
+        callback(translate('virtAsstTitleFullStatus'), status);
+      });
+    });
+
+    // blood sugar and direction
+    configuredPlugin.configureIntentHandler('MetricNow', function (callback, slots, sbx, locale) {
+      entries.list({count: 1}, function(err, records) {
+        var direction;
+        if(translate(records[0].direction)){
+          direction = translate(records[0].direction);
+        } else {
+          direction = records[0].direction;
+        }
+        var status = translate('virtAsstStatus', {
+          params: [
+            sbx.scaleMgdl(records[0].sgv),
+            direction,
+            moment(records[0].date).from(moment(sbx.time))]
+        });
+        
+        callback(translate('virtAsstTitleCurrentBG'), status);
+      });
+    }, ['bg', 'blood glucose', 'number']);
+  
+    // blood sugar delta
+    configuredPlugin.configureIntentHandler('MetricNow', function (callback, slots, sbx, locale) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['locale' is defined but never used.](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)